tcpdf.php

```diff
@@ -9994,6 +9994,9 @@ protected function _enddoc() {
 		$this->_out($o);
 		$this->_out('%%EOF');
 		$this->state = 3; // end-of-doc
+		foreach($this->imagekeys as $file) {
```

Comment:
This code should go into the _destroy() function that is always called after _enddoc().